### PR TITLE
Style bug fix

### LINKED ISSUE #9: make the website mobile responsive , just some bugs
Most of the website is already responsive for mobile, but there are some issues with image display and text behavior. Additionally, the contributors section needs some adjustments. Please fix these bugs .

image
image

### PR DESCRIPTION
**Pull request by RajiAujla**
### Description

This pull request fixes a bug related to fixing navigation menu in the header section on mobile device and contibution section aligment fix.

Fixes #9 (make the website mobile responsive , just some bugs)

### Type of Change

Bug Fix

- Adjusted the CSS for the navigation menu to ensure proper alignment on mobile devices.
- Contribution section content layout fix to handle different sizes more effectively.
- image adjustment to fix the border size on alter screen size.

### How Has This Been Tested?

- Tested on various devices (iPhone 12 Pro, Samsung Galaxy S8+) to ensure the navigation menu and   section adjustment is aligned correctly.
- Used browser developer tools to simulate different screen sizes and verify the changes.
- Ran the full test suite to ensure no existing functionality is broken.

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### Screenshots:

After Fix:
navbar
contribution section

### CODE REVIEW

**Comment by Vikleoboy** [comment]
image
## Task 1
The button is really close to the image. Please adjust the spacing to create some distance between them. This issue only occurs with specific dimensions that are in the picture .

## Task 2
image

Also, reduce the font size for mobile devices to improve readability.

File: components/Navbar.tsx [hunk 1]
```diff
@@ -50,7 +52,7 @@ const Navbar = () => {
 
       {/* Mobile Nav */}
 
-      <section className="navsize:hidden flex  justify-between items-baseline pt-4 px-4 w-full  z-50">
+      <section className="navsize:hidden flex  justify-between items-baseline pt-4 pb-4 px-4 w-full  z-50">
```
Comment (Vikleoboy) [inline components/Navbar.tsx:55]: the text on mobile navbar only says Axiom Make it Axiom books and change its coulor to black plz

**Review by Vikleoboy — CHANGES_REQUESTED** [review]

**Comment by RajiAujla** [comment]
Task and Task 2 has fixed and updated.
Also, the extra space between the sections has been remove and at the end footer bottom padding adjusted too.

Have review and let me know your feedback

**Comment by Vikleoboy** [comment]
Oh soory i merged this but we actully need the space betwwen sections so put back the h-100vh

**Pull request merged**